tracing/src/main/java/com/palantir/tracing/Tracer.java

```diff
@@ -244,7 +244,7 @@ static DetachedSpan detachInternal(@Safe String operation, SpanType type) {
         Optional<String> requestId = getRequestId(maybeCurrentTrace, type);
         return sampled
                 ? new SampledDetachedSpan(operation, type, traceId, requestId, parentSpan)
-                : new UnsampledDetachedSpan(traceId, requestId, parentSpan);
+                : new UnsampledDetachedSpan(traceId, requestId, Optional.empty());
```

Comment:
This is a simplification for readability: parentSpan is always empty here, but it wasn't obvious reading the code.